### ISSUE TITLE
Unable to load Serie A game via PublicWyscoutLoader

### ISSUE THREAD

**Issue by faridmusayev** [ML-KULeuven/socceraction#358]
Hello, 

I receive `StopIteration:` error when trying to load Serie A game with `id = 2576016` from PublicWyscoutLoader.

python version: `3.9.12`
socceraction version: `1.2.3`

Repex is the following:

```# import wyscout public match event data loader from socceraction library
from socceraction.data.wyscout import PublicWyscoutLoader 

# load public wyscout data
wyscout_data = PublicWyscoutLoader()

# retrieve all serie a games
seriea_games = wyscout_data.games(competition_id = 524, season_id = 181248)["game_id"]

# while running iteration I get an error "StopIteration:" at game_id = 2576016
# don't run it by yourself it takes a while
for i in seriea_games:
    df = wyscout_data.players(i)
    df.to_csv('seriea_players/{i}.csv')

# I found a problematic game, checked it separately and received the same error
wyscout_data.players(2576016)

```
I would appreciate you help !

**Comment by probberechts** [comment]
This is a duplicate of #276 and was fixed in d2c282ed9932eff9a5aa8ad4e3c4c73ca6a24b78, which will be included in the next release. In the meantime, you can install the dev version from [test.pypi.org](https://test.pypi.org/project/socceraction/).

**Comment by faridmusayev** [comment]
Great, thank you for help !
